SConstruct

```diff
@@ -663,6 +671,19 @@ if "help" in COMMAND_LINE_TARGETS:
 # *** Read user-configurable options ***
 # **************************************
 
+# Alternate cantera configuration file
+AddOption(
+    "--cantera-conf", dest="cantera-conf", nargs=1, type="string",
+    default="cantera.conf", action="store", help="Cantera configuration file")
+cantera_conf = GetOption("cantera-conf")
+if cantera_conf != "cantera.conf":
+    logger.info(f"Using alternate configuration file '{cantera_conf}'")
+    if Path("cantera.conf").is_file():
+        logger.info("Saving existing 'cantera.conf' as 'cantera.conf.bkp'")
+        sh_copy("cantera.conf", "cantera.conf.bkp")
+    # replace cantera.conf
+    sh_copy(cantera_conf, "cantera.conf")
+
```

Comment:
If you're allowing the option of using an alternate configuration file, why require having a `cantera.conf` file at all? Why not just read the file passed here?

Reply:
Having multiple `cantera.conf` was what made me implement this. Funny enough, I barely ever used it, so I'm starting to seriously question whether this is worth keeping in this PR :smile:

Reply:
That's fine too 😄  I think if it's going to stay, `cantera_conf` should get passed into `Variables()` below, then you don't need to copy the user-specified file into a file called `"cantera.conf"`.

Reply:
gone!